FF1Lib/Classes.cs

```diff
@@ -62,10 +62,10 @@ public enum MpGainOnMaxGain
 
 	public class GameClasses
 	{
-		private List<ClassData> _classes;
-		private GearPermissions _weaponPermissions;
-		private GearPermissions _armorPermissions;
-		private SpellPermissions _spellPermissions;
+		public List<ClassData> _classes;
```

Comment:
don't make these public, they're only there for the internals of gameclasses and they can't be trusted if they're exposed, you should use ArmorPermissions, WeaponPermissions, SpellPermissions and ClassData directly